nixos/doc/manual/release-notes/rl-1609.section.md

```diff
@@ -46,7 +46,7 @@ When upgrading from a previous release, please be aware of the following incompa
 
 Other notable improvements:
 
-- Revamped grsecurity/PaX support. There is now only a single general-purpose distribution kernel and the configuration interface has been streamlined. Desktop users should be able to simply set
+- Revamped grsecurity/PaX support. There is now only a single general-purpose distribution kernel and the configuration interface has been streamlined. Desktop users should be able to set
```

Comment:
I don't think it's a good idea to change past release notes retroactively. Same for other changes below.

Reply:
I can revert them, but I don't see the problem TBH

Reply:
It's not really a problem, I think they should be considered of archived.
Besides, practically no one is going to look at them, anyway.